src/main/java/com/avparamonov/checkers/configurations/SecurityConfig.java

```diff
@@ -37,14 +37,15 @@ protected void configure(final HttpSecurity http) throws Exception {
 
         http
                 .authorizeRequests()
-                .antMatchers(Api.ROOT_PATH).permitAll()
+                .antMatchers(Api.ROOT_PATH, "/webjars/**").permitAll()
                 .antMatchers(Api.ROOT_PATH + Api.V1.LOGIN).permitAll()
                 .antMatchers(Api.ROOT_PATH + Api.V1.REGISTRATION).permitAll()
                 .antMatchers(Api.ROOT_PATH + Api.V1.ADMIN + "/**").hasAuthority("ADMIN")
                 .anyRequest()
-                .authenticated().and().csrf().disable().formLogin()
-                .loginPage(Api.ROOT_PATH + Api.V1.LOGIN).failureUrl(Api.ROOT_PATH + Api.V1.LOGIN + "?error=true")
-                .defaultSuccessUrl(Api.ROOT_PATH + Api.V1.GAME)
+                .authenticated().and().csrf().disable()
+                .formLogin().loginPage(Api.ROOT_PATH + Api.V1.LOGIN)
+                .failureUrl(Api.ROOT_PATH + Api.V1.LOGIN + "?error=true")
+                .defaultSuccessUrl(Api.ROOT_PATH + Api.V1.HOME)
                 .usernameParameter("nickname")
```

Comment:
i'd say it's better to log in using email instead of nickname (easier to remember, instant 'reset password' possibility)